contracts/Overtime/Ticket.sol

```diff
@@ -216,22 +212,22 @@ contract Ticket is OwnedWithInit {
 
     /// @notice withdraw collateral from the ticket
     function withdrawCollateral(address recipient) external onlyAMM {
-        sportsAMM.defaultCollateral().transfer(recipient, sportsAMM.defaultCollateral().balanceOf(address(this)));
+        collateral.transfer(recipient, collateral.balanceOf(address(this)));
```

Comment:
why is safeTransfer not used?

Reply:
I think we used it for SportPositionalMarket without safeTransfer. 
I guess @vladanthales didn't change that and I didn't want to change it in a same branch (same reason as with addressManager)

Reply:
added